packit/cli/__init__.py

```diff
@@ -1,2 +1,18 @@
 # Copyright Contributors to the Packit project.
 # SPDX-License-Identifier: MIT
+
+import click
+
+from packit.cli.config import config  # Import the new config group
+
+
+@click.group()
+def cli():
+    pass
+
+
+# Register the config group.
+cli.add_command(config)
+
+if __name__ == "__main__":
+    cli()
```

Comment:
instead of this, the command should be added [here](https://github.com/packit/packit/blob/69afa2b7e5e8e351c548b5c4f97b42c322658579/packit/cli/packit_base.py#L72)